ltiauthenticator/lti11/handlers.py

```diff
@@ -48,3 +49,30 @@ def post(self):
         )
 
         self.redirect(body_argument)
+
+
+class LTI11ConfigHandler(BaseHandler):
+    """
+    Renders LTI 1.1 configuration file in XML format.
```

Comment:
Could you elaborate a bit in this docstring, perhaps link out to relevant schemas as described in #49?